packages/models/src/contract/interface.ts

```diff
@@ -0,0 +1,12 @@
+import type { ActorURI, CallResponse, MessagePayload } from '../actor'
+
+export type Params = object
```

Comment:
The `Params` is so general that will conflict with one from other module, how about adding a namespace here